ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -78,17 +79,21 @@ const PersonalSignInfo: React.FC = () => {
         padding={2}
         marginBottom={4}
       >
-        <AlertRow
-          alertKey="message"
-          ownerId={currentConfirmation.id}
-          label={t('message')}
-        >
-          <ConfirmInfoRowText
-            text={sanitizeString(
-              hexToText(currentConfirmation.msgParams?.data),
-            )}
-          />
-        </AlertRow>
+        {isSiweSigReq ? (
```

Comment:
Very minor, but `isSIWE` to avoid the abbreviation?

Reply:
done 👍